metadata/org.ehcache/ehcache/3.10.8-jakarta/jni-config.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
- I also removed `com.sun.management.**`, `sun.management.**`,`java.management.**` as requested by https://github.com/oracle/graalvm-reachability-metadata/pull/122#discussion_r1030139343 . 
- @mhalbritter Is there an issue explaining why this has to be done?

Reply:
- Answered at https://github.com/oracle/graalvm-reachability-metadata/pull/124#issuecomment-1327324023.